magento/pwa-studio
# Update styleguide colors & typography

## Description
This PR introduces the first aspects of new design to the Venia Styleguide. It updates colors & typography, and uses them to update the `button` component's presentation.

This PR also renames the `footer` component to `button group`, since it's not really a footer, and updates it to reflect guidance from the new design.

## Related Issue
- PWA-433
- PWA-435

## Acceptance 
@soumya-ashok 
@schensley 

### Verification Stakeholders

### Specification

### Verification Steps
1. Visit `/page/color` and verify the colors are correct
2. Visit `/page/typography` and verify the fonts are correct
3. Visit `/page/button` and verify the appearance and guidelines are correct
4. Visit `/page/button-group` and verify the appearance and guidelines are correct

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
